src/frontend/src/handler/create_table.rs

```diff
@@ -238,4 +337,70 @@ mod tests {
 
         assert_eq!(columns, expected_columns);
     }
+
+    #[test]
+    fn test_bind_primary_key() {
```

Comment:
Can we move these tests to `planner_tests` after the pk of tables is fully implemented?

Reply:
Maybe. It requires some updates on planner test runner, as it always assumes DDLs would succeed and only checks plan or error against the last query statement.